src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -653,7 +679,10 @@ private ITaskItem ReadItem()
         internal sealed class CacheWriter : IDisposable
         {
             private const int InitialStringTableCapacity = 32;
+            private const string ResolvedMetadata = "Resolved";
```

Comment:
I'd suggest adding this to `MetadataKeys` to match the other metadata names.

Reply:
changed